theseus/optimizer/autograd/tests/test_lu_cuda_sparse_backward.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import pytest  # noqa: F401
+import torch
+from torch.autograd import gradcheck
+
+import theseus.core as theseus
+import theseus.optimizer as thoptim
+import theseus.optimizer.linear as thlin
```

Comment:
Following convention from all other test files we should replace with `import theseus as th`.